Formula/conductr-cli.rb

```diff
@@ -0,0 +1,14 @@
+class ConductrCli < Formula
+  desc "The Lightbend ConductR CLI"
+  homepage "https://conductr.lightbend.com"
+  url "https://bintray.com/lightbend/generic/download_file?file_path=conductr-cli-1.0.1-Mac_OS_X-x86_64.zip"
```

Comment:
Awesome. A tarball from the [releases](https://github.com/typesafehub/conductr-cli/releases) should probably be used, then.